.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Pull Request template
+Please, go through these steps before you submit a PR.
+
+1. This repository follows semantic versioning convention, therefore each PR title/commit message must follow convention: `<type>(<scope>): <subject>`. 
```

Comment:
thanks @grzesuav 

Will it be possible to add a tip or two about `subject` as well.
For example, following are valid:
- `#issueno short title` or
- `short title`

Reply:
refer to this [commit](https://github.com/AmitKumarDas/metac/pull/132/commits/7678e907a4756e70584a1beee1792f2923f7bd60)

We may ignore if above is not valid as per semantic conventions

Reply:
will add this later today, good idea :+1:

Reply:
added